kod/object/item.kod

```diff
@@ -633,6 +690,46 @@ messages:
       return FALSE;
    }
 
+   GetMagicRarityGradeCount()
+   "Checks plItem_attributes for HasRarity() = TRUE."
+   "For each attribute with HasRarity = TRUE,  add 1 to count"
+   "Return either viRarity or count, whichever is larger"
+   "This ensures the higher rarity is chosen if in the case "
+   "of competing rarities for an item."
```

Comment:
Put periods at the ends of sentences or this string will look like a mess in admin commands.

Reply:
Fixed.  I did not know the impact of this docstring on admin commands.  I have adjusted the spacing and periods accordingly.